### PR TITLE
Task/influx quix updates

### PR DESCRIPTION
**Pull request by SteveRosam**
Adding alternative downsampling examples - Using Quix.

This PR includes changes to:

 - Dedicated
 - Cloud(TSM)
 - OSS
 - Clustered

Quix is collaborating with Influxdata to act as a pure Python alternative to Kapacitor+Flux.

The audience is folks who:

Know Python already and/or don't want to learn a DSL, and possibly those who want to process data more "Pythonically" rather than mixing SQL syntax and Python.
FYI: Quix is both a runtime platform (Quix Cloud) and data processor (Quix Streams Python library). The aim of the articles is to show how downsampling can be done in a continuous real-time process using pure Python.

[x] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
[x] Rebased/mergeableAdding alternative downsampling examples - Using Quix.

### CODE REVIEW

File: content/influxdb/cloud/tools/downsample-data-quix.md [hunk 1]
```diff
@@ -22,24 +22,22 @@ in [Quix Cloud](https://quix.io/) with a free trial.
...
 it to InfluxDB to reduce the overall disk usage as data collects over time.
 
-This guide walks through the process of creating a series of Python services that ingest from InfluxDB v2, downsample and publish
-data to InfluxDB v3. By aggregating data within windows of time, then storing the aggregate values back to InfluxDB, you can reduce 
+This guide walks through the process of creating a series of Python services that ingest from an InfluxDB v2 bucket, downsample and publish the data to another InfluxDB v2 bucket. 
+By aggregating data within windows of time, then storing the aggregate values back to InfluxDB, you can reduce 
 disk usage and costs over time.
 
-The guide uses Python and InfluxDB v2 and v3 Python client libraries and can be run locally or deployed within Quix Cloud with a free trial. It assumes you have setup a Python project and virtual environment.
+The guide uses the InfluxDB v2 and QuixStreams Python client libraries and can be run locally or deployed within Quix Cloud with a free trial. It assumes you have setup a Python project and virtual environment.
```
Comment (tbedford) [inline content/influxdb/cloud/tools/downsample-data-quix.md]: Quix Streams

File: content/influxdb/v2/tools/downsample-data-quix.md [hunk 2]
```diff
@@ -12,26 +12,24 @@ weight: 122
...
+it to InfluxDB to reduce the overall disk usage as data collects over time.
 
-This guide walks through the process of creating a series of Python services that ingest from InfluxDB v2, downsample and publish
-data to InfluxDB v3. By aggregating data within windows of time, then storing the aggregate values back to InfluxDB, you can reduce 
+This guide walks through the process of creating a series of Python services that ingest from an InfluxDB v2 bucket, downsample and publish the data to another InfluxDB v2 bucket. 
+By aggregating data within windows of time, then storing the aggregate values back to InfluxDB, you can reduce 
 disk usage and costs over time.
 
-The guide uses Python and InfluxDB v2 and v3 Python client libraries and can be run locally or deployed within Quix Cloud with a free trial. It assumes you have setup a Python project and virtual environment.
+The guide uses the InfluxDB v2 and QuixStreams Python client libraries and can be run locally or deployed within Quix Cloud with a free trial. It assumes you have setup a Python project and virtual environment.
```
Comment (tbedford) [inline content/influxdb/v2/tools/downsample-data-quix.md]: Quix Streams

File: content/influxdb/v2/process-data/common-tasks/downsample-data.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ menu:
     name: Downsample data
     parent: Common tasks
 weight: 201
-influxdb/v2/tags: [tasks]
+influxdb/cloud/tags: [tasks]
```
Comment (tbedford) [inline content/influxdb/v2/process-data/common-tasks/downsample-data.md]: Just checking, this is the v2 docs, but you're using influxdb/cloud/tags.

File: content/influxdb/v2/process-data/common-tasks/downsample-data.md [hunk 4]
```diff
@@ -66,7 +66,7 @@ Again, this is a very basic example, but it should provide you with a foundation
 to build more complex downsampling tasks.
 
 ## Add your task
-Once your task is ready, see [Create a task](/influxdb/v2/process-data/manage-tasks/create-task) for information about adding it to InfluxDB.
+Once your task is ready, see [Create a task](/influxdb/cloud/process-data/manage-tasks/create-task) for information about adding it to InfluxDB.
```
Comment (tbedford) [inline content/influxdb/v2/process-data/common-tasks/downsample-data.md]: Also in v2 docs, but you changed the link to cloud. Just checking.

File: content/influxdb/cloud-dedicated/process-data/downsample/downsample-quix.md [hunk 5]
```diff
@@ -32,7 +32,7 @@ This guide also assumes you have already
 
 The following diagram illustrates how data is passed between processes as it is downsampled:
 
-{{< html-diagram/quix-downsample-pipeline >}}
+{{< html-diagram/influxdb-v3-quix-downsample-pipeline >}}
 
 {{% note %}}
 It is usually more efficient to write raw data directly to Kafka rather than
```
Comment (tbedford) [inline content/influxdb/cloud-dedicated/process-data/downsample/downsample-quix.md:38]: Further down on line 68, dependencies list should match what's actually pip installed.

Comment (tbedford) [inline content/influxdb/cloud-dedicated/process-data/downsample/downsample-quix.md:38]: Line 102: you could simplify as JSON is default, so you don't need the serializers/deserializers.

image

File: content/influxdb/cloud/tools/downsample-data-quix.md [hunk 6]
```diff
@@ -22,24 +22,22 @@ in [Quix Cloud](https://quix.io/) with a free trial.
 A common practice when processing high volume data is to downsample it before comitting
```
Comment (tbedford) [inline content/influxdb/cloud/tools/downsample-data-quix.md:22]: I don't see this topic where I expected it "cloud/tools". Very confusing.

image

**Review by tbedford — CHANGES_REQUESTED** [review]
Seems to all render fine. The main issue was I could not see `cloud/tools/downsample-data-quix.md` content in the nav where I expected it. I had selected InfluxDB Cloud(TSM) from the product selector. 

Note that my comments on the installable dependencies list, and also regarding JSON being the default serializer/deserializer apply across all files modified - but I only commented on the one file.

File: content/influxdb/v2/process-data/common-tasks/downsample-data.md (hunk 3 above)
Comment (SteveRosam) [reply content/influxdb/v2/process-data/common-tasks/downsample-data.md]: Yep this was unintentional

File: content/influxdb/v2/process-data/common-tasks/downsample-data.md (hunk 4 above)
Comment (SteveRosam) [reply content/influxdb/v2/process-data/common-tasks/downsample-data.md]: Reverted.

File: content/influxdb/cloud-dedicated/process-data/downsample/downsample-quix.md (hunk 5 above)
Comment (SteveRosam) [reply content/influxdb/cloud-dedicated/process-data/downsample/downsample-quix.md:38]: @tbedford not sure what you're referring to here

File: content/influxdb/cloud/tools/downsample-data-quix.md (hunk 6 above)
Comment (SteveRosam) [reply content/influxdb/cloud/tools/downsample-data-quix.md:22]: hopefully dealt with

**Review by tbedford — APPROVED** [review]
LGTM!

**Pull request merged**